core/src/main/java/io/substrait/relation/VirtualTableScan.java

```diff
@@ -10,8 +9,6 @@
 @Value.Immutable
 public abstract class VirtualTableScan extends AbstractReadRel {
 
-  public abstract List<String> getDfsNames();
```

Comment:
Removing this is a breaking change as it removes the `addAllDfsNames` method on the `ImmutableVirtualTableScanBuilder`, but that's reasonable I think.

It is causing a failure in `SubstraitRelVisitor` which uses the `addAllDfsNames`, but we should be able to replace that with a call to `initialSchema`.

Reply:
Ah right, turns out there were two tests that had similar issues as well, should be all fixed by https://github.com/substrait-io/substrait-java/pull/272/commits/14b7a1b1a8efa7b9706d6e1fa1f5a5930078b05f